conf/polarion_tools.yaml

```diff
@@ -66,6 +66,7 @@ docstrings:
         # default for manual mark is 'notautomated'
         # @pytest.mark.manual('manualonly') to set 'manualonly'
         linkedWorkItems: "@pytest.mark.requirements"
+        customerscenario: "@pytest.mark.requirements.customer_stories"
```

Comment:
Not possible to set meta with these markers, evaluating if its worth supporting this.  Could also be set via BZ marker if by resolving items attached to the BZ.